internal/rollup_bundle/rollup_bundle.bzl

```diff
@@ -123,11 +123,11 @@ def run_uglify(ctx, input, output, debug = False, comments = True):
   )
 
 def _rollup_bundle(ctx):
-  rollup_config = write_rollup_config(ctx)
-  run_rollup(ctx, collect_es6_sources(ctx), rollup_config, ctx.outputs.build_es6)
-  run_tsc(ctx, ctx.outputs.build_es6, ctx.outputs.build_es5)
-  run_uglify(ctx, ctx.outputs.build_es5, ctx.outputs.build_es5_min)
-  run_uglify(ctx, ctx.outputs.build_es5, ctx.outputs.build_es5_min_debug, debug = True)
+  rollup_config = _write_rollup_config(ctx)
+  _run_rollup(ctx, collect_es6_sources(ctx), rollup_config, ctx.outputs.build_es6)
```

Comment:
angular calls these functions from `ng_rollup_bundle` - could you check this PR against angular/angular before we merge it?